api/standard/clause_8_ogcapi_processes.adoc

```diff
@@ -1,18 +1,18 @@
 [[processes-profile]]
-== Routing API as a profile of OGC API Processes
+== Routing API as a profile of _OGC API - Processes_
 
-This clause specifies a second API option for a Routing API that supports more-or-less the same capabilities as the Routing API defined in the previous clause and re-uses the Route Exchange Model. However, it is based on the OGC API Processes standards and defined as a profile of that standard.
+This clause specifies a second API option for a Routing API that supports more-or-less the same capabilities as the Routing API defined in the previous clause and re-uses the Route Exchange Model. However, it is based on the _OGC API - Processes_ standards and defined as a profile of that standard.
 
-#TODO: Do we want to keep the OGC API Processes option? If yes, we need to align with the latest draft. This is an old version based on an outdated draft of OGC API Processes. As discussed in the meeting on 2021-01-12, the preference is for a single API with explicit Route resources as described in <<routing-api>>. We will analyse, if the changes to OGC API Processes in recent month support that we can basically merge the two approaches without complicating the API. If that is not possible, an OGC API Processes profile for routing could be specified as a separate document, if there is interest, potentially by another group.#
+#TODO: Do we want to keep the _OGC API - Processes_ option? (Yes!) If yes, we need to align with the latest draft (Done). This is an old version based on an outdated draft of _OGC API - Processes_. As discussed in the meeting on 2021-01-12, the preference is for a single API with explicit Route resources as described in <<routing-api>>. We will analyse, if the changes to _OGC API - Processes_ in recent month support that we can basically merge the two approaches without complicating the API. If that is not possible, an _OGC API - Processes_ profile for routing could be specified as a separate document, if there is interest, potentially by another group.#
```

Comment:
If its done, maybe remove this part.

Reply:
We can remove the paragraph. @jeffharrison will create a new issue on this aspect capturing the discussion in the 2021-02-09 call. This PR will "just" be about updating the chapter to the current Processes draft.